packages/extension-ui/src/components/Address.tsx

```diff
@@ -254,30 +255,18 @@ function Address({
             >
               {_ellipsisName(formatted) || _ellipsisName(address) || t('<unknown>')}
             </div>
-            {/* TODO: this logic will be reused */}
-            {/* <CopyToClipboard text={(formatted && formatted) || ''}>
-              <FontAwesomeIcon
-                className='copyIcon'
-                icon={faCopy}
-                onClick={_onCopy}
-                size='sm'
-                title={t('copy address')}
-              />
-            </CopyToClipboard>
-            {(actions || showVisibilityAction) && (
-              <FontAwesomeIcon
-                className={isHidden ? 'hiddenIcon' : 'visibleIcon'}
-                icon={isHidden ? faEyeSlash : faEye}
+            {isHidden && (
+              <img
+                className='hiddenIcon'
                 onClick={_toggleVisibility}
-                size='sm'
-                title={t('account visibility')}
+                src={viewOff}
               />
-            )} */}
+            )}
           </div>
         </div>
         {actions && (
           <>
-            <Link to='/account/edit-menu'>
+            <Link to={`/account/edit-menu/${address || ''}/${isExternal ? 'true' : 'false'}`}>
```

Comment:
Can we have `/account/edit-menu/true/` if there is no address? is this valid?
Also to be honest it's not a good practice to pass  `true/false` in params. If you need to pass such data it should be a query param like `/account/edit-menu/${address || ''}/${isExternal ? '?isExternal=true' : '?isExternal=false'}`